content-scripts/inject/module.js

```diff
@@ -36,6 +36,10 @@ const page = {
 
   fireEvent(info) {
     if (info.addonId) {
+      if (info.name === "enable") {
+        runAddonUserscripts({ ...info, enabledLate: true });
+        return;
+      }
```

Comment:
I'd prefer a dedicated message for this - no need to make the background page directly call this.